packages/webapp/src/components/Modals/BulkSensorUploadModal/index.jsx

```diff
@@ -23,6 +23,7 @@ export default function BulkSensorUploadModal({
   onTemplateDownloadClick,
   uploadErrorMessage,
   uploadErrorLink,
+  errorTypeCode,
```

Comment:
This should also use the same object of enums

Reply:
Done